tcms/utils/tests/test_assign_permissions.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth.models import Group, Permission
+from django.test import TestCase
+
+from tcms.utils.permissions import assign_default_group_permissions
+
+
+class TestAssignDefaultGroupPermissions(TestCase):
+    """Test utils.permissions.assign_default_group_permissions()"""
+
+    def setUp(self):
+        self.admin = Group.objects.get(name='Administrator')
+        self.admin.permissions.all().delete()
+        self.tester = Group.objects.get(name='Tester')
+        self.tester.permissions.all().delete()
+
+    def test_administrator_has_all_permissions(self):
+        assign_default_group_permissions()
+        self.assertEqual(Permission.objects.all().count(),
+                         self.admin.permissions.count())
+
+    def test_tester_has_kiwi_apps_permissions(self):
+        assign_default_group_permissions()
+        for app_name in ['bugs', 'django_comments', 'linkreference', 'management',
```

Comment:
See failure in https://github.com/kiwitcms/Kiwi/pull/2018/checks?check_run_id=1305514506.  The `bugs` app can be turned off and when it is off there are no permissions with that name. See bugs/tests/ for hints.